__tests__/Config.test.js

```diff
@@ -97,6 +98,10 @@ describe('Config', () => {
       expect(Config.sinopiaUrl).toEqual('https://sinopia.foo')
     })
 
+    it('sinopia env overrides static value for display', () => {
```

Comment:
To be consistent with the other values in here, you should add a static test above too for when the env var is undefined.